l10n_it_fatturapa_in_rc/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Sergio Corato (https://efatto.it)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'ITA - Fattura elettronica - Reverse charge',
+    "summary": "Modulo di collegamento tra e-fattura in acquisto e reverse"
+               " charge",
```

Comment:
Sarebbe meglio mantenere uniformità con il modulo
https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_reverse_charge

Inoltre qui in summary è stato usato "Modulo di collegamento", in DESCRIPTION.rst "Modulo ponte", sarebbero da uniformare.